includes/User.php

```diff
@@ -429,7 +429,11 @@ public function saveToCache() {
 	}
 
 	private function getCacheKey() {
-		return wfMemcKey('user', 'id', $this->mId);
+		return self::getCacheKeyById( $this->mId );
+	}
+
```

Comment:
Needed a static method to return the cache key so there could be a static method to clear the user cache.  Updated the instance method ```getCacheKey``` to use this static method to keep everything aligned.